src/transaction/log_manager.c

```diff
@@ -10214,6 +10219,482 @@ logtb_tran_update_stats_online_index_rb (THREAD_ENTRY * thread_p, void *data, vo
   return error_code;
 }
 
+int
+cdc_get_lsa (THREAD_ENTRY * thread_p, time_t * time, LOG_LSA * start_lsa)
+{
+  /*
+   * 1. get volume list
+   * 2. get fpage from each volume 
+   * 3. get commit/abort/ha_dummy_server_state which contains time from fpage 
+   * */
+  int begin = log_Gl.hdr.last_deleted_arv_num;
+  int end = log_Gl.hdr.nxarv_num - 1;
+  char arv_name[PATH_MAX];
+  LOG_ARV_HEADER *arv_hdr;
+  int num_arvs = end - begin;
+
+  time_t active_start_time = 0;
+  time_t archive_start_time = 0;
+  int target_arv_num = -1;
+
+  LOG_LSA ret_lsa;
+  bool is_found = false;
+
+  int error = NO_ERROR;
+
+  /*
+   * 1. traverse from the latest log volume 
+   * 2. when num_arvs > 0, no logic to handle the active log volume 
+   * 3. check condition when i = begin while finding target_arv_num 
+   */
+
+  /*AT first, compare the time in active log volume. */
+  error = get_start_point_from_file (thread_p, -1, &ret_lsa, &active_start_time);
+  if (error == ER_FAILED)
+    {
+      return error;
+    }
+  else if (error == ER_CDC_LSA_NOT_FOUND)
+    {
+      /* rare case : can not find LOG that contains time information. */
```

Comment:
Even if the active log didn't contain time information, we can try to rummage the archives, can't we?